simtools/model/array_model.py

```diff
@@ -108,6 +108,11 @@ def _initialize(self, site, array_elements_config):
             array_elements = self._get_array_elements_from_list(
                 site_model.get_array_elements_for_layout(self.layout_name)
             )
+        if array_elements == {}:
+            raise ValueError(
+                "No array elements found. "
+                "Did you provide a valid layout name or list of telescopes?"
```

Comment:
I don't like the 'you' in the message - it implies that the user is a person. Maybe "Possibly missing valid layout name or missing telescope list"?

Reply:
No problem.